providers/configcat/src/test/resources/features.json

```diff
@@ -1,88 +1,217 @@
 {
+  "p": {
+    "s": "80xCU/SlDz1lCiWFaxIBjyJeJecWjq46T4eu6GtozkM="
+  },
   "f": {
     "enabledFeature": {
-      "v": true
+      "t": 0,
+      "v": {
+        "b": true
+      }
     },
     "intSetting": {
-      "v": 5
+      "t": 2,
+      "v": {
+        "i": 5
+      }
     },
     "doubleSetting": {
-      "v": 3.14
+      "t": 3,
+      "v": {
+        "d": 3.14
+      }
     },
     "stringSetting": {
-      "v": "test"
+      "t": 1,
+      "v": {
+        "s": "test"
+      }
     },
     "userIdMatching": {
-      "v": false,
+      "t": 0,
       "r": [
         {
-          "o": 0,
-          "a": "Identifier",
-          "t": 2,
-          "c": "@matching.com",
-          "v": true
+          "c": [
+            {
+              "u": {
+                "a": "Identifier",
+                "c": 2,
+                "l": [
+                  "@matching.com"
+                ]
+              }
+            }
+          ],
+          "p": [
```

Comment:
You don't need a percentage options array here, but you need a return value for the conditions.  This is true for the other flags in this JSON as well, where a percentage array was added.

The `userIdMatching` should look like this:

```
"userIdMatching": {
      "t": 0,
      "r": [
        {
          "c": [
            {
              "u": {
                "a": "Identifier",
                "c": 2,
                "l": [
                  "@matching.com"
                ]
              }
            }
          ],
          "s": {
              "v": {
                "b": true
              },
              "i": "bcfb84a7"
            }
        }
      ],
      "v": {
        "b": false
      },
      "i": "430bded3"
    }
```